examples/multi_gpu/README.md

```diff
@@ -2,16 +2,23 @@
 
 ## Examples with NVIDIA GPUs
 
+### Examples with CuGraph
+
+| Example                                                                        | Scalability | Description                                                                              |
+| ------------------------------------------------------------------------------ | ----------- | ---------------------------------------------------------------------------------------- |
+| [`ogbn_train_cugraph.py`](./ogbn_train_cugraph.py)                             | single-node | Single Node Multi GPU Example for `ogbn_train.py` using [CuGraph](...).                  |
+| [`papers100m_gcn_cugraph_multinode.py`](./papers100m_gcn_cugraph_multinode.py) | multi-node  | Example for training GNNs on a homogeneous graph on multiple nodes using [CuGraph](...). |
```

Comment:
Can we update these `[CuGraph](...)`? Once updated, this PR is ready, and feel free to merge it :)

(You can add links to blog posts, articles, etc. (if there is) in a follow-up. I'd just like to make sure users are attracted enough to cugraph to give it a try when landing on this page :))